src/components/Geogebra/GeogebraViews.js

```diff
@@ -74,15 +75,43 @@ export default class {
             }
         });
 
+        api.service('elements').on('removed', (element) => {
+            const pos = this.workshopIds.indexOf(element.workshop);
```

Comment:
`Array` has `includes` method now - we don't need to rely on `indexOf` to determine if it contains an element
https://developer.mozilla.org/pl/docs/Web/JavaScript/Referencje/Obiekty/Array/includes

Reply:
I need the position in array, not only Boolean.